variables.tf

```diff
@@ -394,38 +372,38 @@ variable "rules_with_source_cidr_blocks" {
     cidr_blocks = list(string)
   }))
   default = {
-    "kong-ingress-proxy-http" = {
+    "kong-ingress-proxy-https" = {
```

Comment:
what wa the reason for changing this from `http` to `https`?

Reply:
Possibly left over from testing. Do we want to keep the default http? Then override with https rules for the ecs module?

Reply:
default of https is fine